bloom.go

```diff
@@ -23,7 +23,7 @@ func New(n uint, fp float64, bitSet BitSetProvider) *BloomFilter {
 }
 
 func estimateParameters(n uint, p float64) (uint, uint) {
-	m := math.Ceil((float64(n) * math.Log(p)) / math.Log(1.0 / (math.Pow(2.0, math.Ln2))))
+	m := math.Ceil((float64(n) * math.Log(p)) / math.Log(1.0/(math.Pow(2.0, math.Ln2))))
 	k := math.Log(2.0) * m / float64(n)
```

Comment:
These lines can be:

```go
m := math.Ceil(float64(n) * math.Log(p) / math.Log(1.0/math.Pow(2.0, math.Ln2)))
k := math.Round(math.Ln2 * m / float64(n))
```

(removed some unnecessary parens, added rounding instead of truncation on k, per http://hur.st/bloomfilter).

Strictly speaking the dot-zeros are not necessary either, but they're certainly clearer.